__tests__/app.test.js

```diff
@@ -23,19 +25,55 @@ describe("GET /api/topics", () => {
   });
 });
 
-// 2. GET /api
+// // 2. GET /api
 
 describe("GET /api", () => {
   test("1. Returns the endpoints.json object describing all the available endpoints on your API", () => {
     return request(app)
       .get("/api")
       .then((response) => {
         expect(response.body).toBeInstanceOf(Object);
-        Object.keys(response.body).forEach((endpoint) => {
-          expect(response.body[endpoint]).toHaveProperty("description", expect.any(String));
-          expect(response.body[endpoint]).toHaveProperty("queries", expect.any(Array));
-          expect(response.body[endpoint]).toHaveProperty("exampleResponse", expect.any(Object));
+        expect(response.body.endpoints).toMatchObject(endpointsFile);
+      });
+  });
+});
+
+// 3. GET /api/articles/:article_id
+
+describe("GET /api/articles/:article_id", () => {
+  test("1. GET: 200 sends an article object by its id with the correct properties", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article).toEqual({
```

Comment:
think about using a flexible matcher eg expect.objectContaining or .toMatchObject. This allows this endpoint to be extended at a future time